### ISSUE TITLE
 Jobs don't run

### ISSUE THREAD

**Issue by mac3388** [freescout-helpdesk/freescout#3608]
Hello, I have just installed Freescout via Softaculous. 

I can't start the jobs

In queue-jobs  the following appears

sh: 1: : Permission denied

 What authorization is meant here?

-->

PHP version: 8.2.13
FreeScout version: 1.8.112
Database: MySQL 
Are you using CloudFlare: No

**Comment by freescout-helpdesk** [comment]
Unfortunately we don't have any ideas - it need to be checked on your server.

**Comment by mac3388** [comment]
ok that's no problem. should I send you the access data?

**Comment by freescout-helpdesk** [comment]
You may need to find someone: https://github.com/freescout-helpdesk/freescout/wiki/Hire-Developer

**Comment by raramuridesign** [comment]
@freescout-helpdesk I hope all is well.

We have just picked up this as an issue on our instance - 
I cannot find any other reference in the issues which relates to this, and I find this highly unusual.
We have recently migrated to a new server, however the only thing not working is email fetching and sending.

If we do a connection test on the 3 different email boxes, they all send test emails without issues and they conenct to the email boxes using IMAP without issues.

My test emails have not moved in the email log

![Image](https://github.com/user-attachments/assets/a1952d7d-8b57-4720-99e2-bcc54d77c683)

Here is a view of the background jobs, which also seem stuck.

![Image](https://github.com/user-attachments/assets/66936f30-6f79-47f0-8907-17304873f704)

Would you be able to review the error logs below and let us know what this issue could be - Thanks.

```
Setup
App Version | 1.8.173
Mar 31, 2025 21:46
Africa/Johannesburg (GMT+0200)
HTTPS
Mysql (10.11.11-MariaDB-ubu2204)
Apache/2.4.63 (Unix) OpenSSL/3.0.2
PHP 8.3.17
256M / 256M
```

Server logs, are showing no errors relating to this

_System logs from Freescout, show the following_ 

**Que Logs**

![Image](https://github.com/user-attachments/assets/00f91374-1c02-4dcf-ba5c-bde922d17fbd)

**Fetch email logs**

![Image](https://github.com/user-attachments/assets/f13c9a3b-72fb-44ee-909b-f9b27a71754e)

** Laravel Logs ** 
_I am not sure this is related, but added this here anyway. (note we have cleared the cache a few times, and I checked the file system and this is not there. So it might be unrelated.)_

```
file_put_contents(/home/projects/webapps/app-helpdesk/public/storage/framework/cache/data/ee/2f/ee2f842aa7bb1f53edf3a2ed2c09a1807ffa6c90): Failed to open stream: Permission denied {"exception":"[object] (ErrorException(code: 0): file_put_contents(/home/projects/webapps/app-helpdesk/public/storage/framework/cache/data/ee/2f/ee2f842aa7bb1f53edf3a2ed2c09a1807ffa6c90): Failed to open stream: Permission denied at /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:122)
[stacktrace]
#0 [internal function]: Illuminate\\Foundation\\Bootstrap\\HandleExceptions->handleError()
#1 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(122): file_put_contents()
#2 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Cache/FileStore.php(64): Illuminate\\Filesystem\\Filesystem->put()
#3 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Cache/FileStore.php(119): Illuminate\\Cache\\FileStore->put()
#4 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Cache/Repository.php(303): Illuminate\\Cache\\FileStore->forever()
#5 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Cache/CacheManager.php(304): Illuminate\\Cache\\Repository->forever()
#6 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(221): Illuminate\\Cache\\CacheManager->__call()
#7 /home/projects/webapps/app-helpdesk/public/app/Misc/Helper.php(1213): Illuminate\\Support\\Facades\\Facade::__callStatic()
#8 /home/projects/webapps/app-helpdesk/public/app/Console/Commands/ClearCache.php(76): App\\Misc\\Helper::queueWorkerRestart()
#9 [internal function]: App\\Console\\Commands\\ClearCache->handle()
#10 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Container/BoundMethod.php(28): call_user_func_array()
#11 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Container/BoundMethod.php(87): Illuminate\\Container\\BoundMethod::Illuminate\\Container\\{closure}()
#12 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Container/BoundMethod.php(27): Illuminate\\Container\\BoundMethod::callBoundMethod()
#13 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Container/Container.php(549): Illuminate\\Container\\BoundMethod::call()
#14 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Console/Command.php(183): Illuminate\\Container\\Container->call()
#15 /home/projects/webapps/app-helpdesk/public/overrides/symfony/console/Command/Command.php(255): Illuminate\\Console\\Command->execute()
#16 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Console/Command.php(169): Symfony\\Component\\Console\\Command\\Command->run()
#17 /home/projects/webapps/app-helpdesk/public/overrides/symfony/console/Application.php(992): Illuminate\\Console\\Command->run()
#18 /home/projects/webapps/app-helpdesk/public/overrides/symfony/console/Application.php(255): Symfony\\Component\\Console\\Application->doRunCommand()
#19 /home/projects/webapps/app-helpdesk/public/overrides/symfony/console/Application.php(148): Symfony\\Component\\Console\\Application->doRun()
#20 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Console/Application.php(88): Symfony\\Component\\Console\\Application->run()
#21 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Console/Application.php(177): Illuminate\\Console\\Application->run()
#22 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(249): Illuminate\\Console\\Application->call()
#23 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(221): Illuminate\\Foundation\\Console\\Kernel->call()
#24 /home/projects/webapps/app-helpdesk/public/app/Http/Controllers/SystemController.php(286): Illuminate\\Support\\Facades\\Facade::__callStatic()
#25 [internal function]: App\\Http\\Controllers\\SystemController->toolsExecute()
#26 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Routing/Controller.php(54): call_user_func_array()
#27 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(45): Illuminate\\Routing\\Controller->callAction()
#28 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Route.php(211): Illuminate\\Routing\\ControllerDispatcher->dispatch()
#29 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Route.php(169): Illuminate\\Routing\\Route->runController()
#30 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Routing/Router.php(658): Illuminate\\Routing\\Route->run()
#31 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(30): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
#32 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/CheckRole.php(26): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#33 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\CheckRole->handle()
#34 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#35 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/CustomHandle.php(27): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#36 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\CustomHandle->handle()
#37 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#38 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/FrameGuard.php(14): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#39 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\FrameGuard->handle()
#40 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#41 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/LogoutIfDeleted.php(28): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#42 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\LogoutIfDeleted->handle()
#43 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#44 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/Localize.php(27): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#45 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\Localize->handle()
#46 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#47 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/HttpsRedirect.php(46): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#48 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\HttpsRedirect->handle()
#49 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#50 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(41): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#51 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle()
#52 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#53 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Auth/Middleware/Authenticate.php(45): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#54 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Auth\\Middleware\\Authenticate->handle()
#55 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#56 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(67): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#57 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle()
#58 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#59 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#60 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle()
#61 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#62 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/TokenAuth.php(32): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#63 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\TokenAuth->handle()
#64 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#65 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(63): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#66 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Session\\Middleware\\StartSession->handle()
#67 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#68 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#69 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle()
#70 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#71 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(61): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#72 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle()
#73 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#74 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#75 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Routing/Router.php(656): Illuminate\\Pipeline\\Pipeline->then()
#76 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\\Routing\\Router->runRouteWithinStack()
#77 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Routing/Router.php(601): Illuminate\\Routing\\Router->runRoute()
#78 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Routing/Router.php(590): Illuminate\\Routing\\Router->dispatchToRoute()
#79 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(176): Illuminate\\Routing\\Router->dispatch()
#80 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(30): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
#81 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/TerminateHandler.php(12): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#82 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\TerminateHandler->handle()
#83 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#84 /home/projects/webapps/app-helpdesk/public/app/Http/Middleware/ResponseHeaders.php(11): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#85 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): App\\Http\\Middleware\\ResponseHeaders->handle()
#86 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#87 /home/projects/webapps/app-helpdesk/public/vendor/fideloper/proxy/src/TrustProxies.php(56): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#88 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Fideloper\\Proxy\\TrustProxies->handle()
#89 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#90 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(30): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#91 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#92 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#93 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(30): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#94 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#95 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#96 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#97 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
#98 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#99 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(46): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#100 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\\Foundation\\Http\\Middleware\\CheckForMaintenanceMode->handle()
#101 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#102 /home/projects/webapps/app-helpdesk/public/overrides/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\\Routing\\Pipeline->Illuminate\\Routing\\{closure}()
#103 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(151): Illuminate\\Pipeline\\Pipeline->then()
#104 /home/projects/webapps/app-helpdesk/public/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(116): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#105 /home/projects/webapps/app-helpdesk/public/public/index.php(82): Illuminate\\Foundation\\Http\\Kernel->handle()
#106 {main}
"}                  
```

**Comment by raramuridesign** [comment]
@freescout-helpdesk I have managed to find out what the issue is after several hours of troubleshooting.

We were migrating the project from one server to another and using a backup / restore method.
This worked well and there was no data loss. 

**This error:** sh: 1: : Permission denied

Was a direct result of running the freescout cronjobs on different servers. 
Essentially the old cronjob was stealing the emails and the new server crons could not keep up with this, and could not find the emails, and this resulted in the permission denied. When we disabled the cronjob on the old server - then everything started working again. 

If we executed the cronjob manually on the server, it fetched the emails, but automating this resulted in the error above.

Just commenting so users are aware of the potential issue.
M
